grcs/0020-knowledge-graph.md

```diff
@@ -373,100 +369,103 @@ An op represents a single atomic operation that produces or modifies knowledge.
 ```proto
 message Op {
   OpType type = 1;
-  Triple triple = 2;
-  Entity entity = 3;
-  Relation relation = 4;
-  repeated Triple triples = 5;
+  Entity entity = 2;
+  Relation relation = 3;
+  Property property = 4;
 }
-```
+``` 
 
 ```proto
 message Relation {
```

Comment:
Byron: Should we have a to_property so people can reference specific properties on an entity?

Yeah I actually thought of some use cases where we would likely want this, so I think it's probably worth adding.